support/client/lib/vwf/model/sound.js

```diff
@@ -283,6 +287,10 @@ define( [ "module", "vwf/model" ], function( module, model ) {
                 this.playOnLoad = soundDefinition.playOnLoad;
             }
 
+            if (this.soundDefinition.useTextToSpeech !== undefined) {
```

Comment:
It is a VWF convention to space things as such: if ( this.soundDefinition.useTextToSpeech !== undefined )

This is a comment that could be applied throughout but I'll save the spam and just say to look out for it :)